lib/model/folder_sendrecv.go

```diff
@@ -2086,14 +2085,13 @@ func (f *sendReceiveFolder) scanIfItemChanged(name string, stat fs.FileInfo, ite
 	if err != nil {
 		return fmt.Errorf("comparing item on disk to db: %w", err)
 	}
-
 	if !statItem.IsEquivalentOptional(item, protocol.FileInfoComparison{
 		ModTimeWindow:   f.modTimeWindow,
 		IgnorePerms:     f.IgnorePerms,
 		IgnoreBlocks:    true,
 		IgnoreFlags:     protocol.LocalAllFlags,
 		IgnoreOwnership: !f.SyncOwnership,
-		IgnoreXattrs:    !f.SyncXattrs,
+		IgnoreXattrs:    fromDelete || !f.SyncXattrs,
```

Comment:
From what I can see this is the only noteworthy significant change.  All other changes in this PR are just "argument-passing."

The only concerns I see here is how could you reliably test this change on different platforms and _what set of cases (in addition to #9371) would you test this against?_

I would look into what test files use this block in their test, and what gh issues those tests were related to, if you are looking to speed this up.

I'd also recommend keeping up your vigor with documenting your work (taking screenshots, recordings, etc), since that seems to have gotten a positive response on your other issues.